includes/compat/class-log.php

```diff
@@ -251,11 +253,16 @@ public function file_download_log_get_post_meta( $value, $object_id, $meta_key,
 						$key = 'api_key';
 						break;
 					case 'user':
-						$key = 'user_id';
+						$key = 'customer_id';
+
 						break;
 				}
 
 				$value = $file_download_log->{$key};
+				if ( 'user' === $meta_key ) {
```

Comment:
```suggestion
				if ( 'user_id' === $key ) {
```

Because in the switch statement above, `$key` `user` is converted to `user_id`.